<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="draw3DGraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3D Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="navigation">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Navigation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chartButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fitting Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>128</width>
      <height>85</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Original City</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="startAddress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>152</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination City</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="endAddress">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>136</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Taxi Cost</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="taxiCost">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>112</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Estimated time</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="estimateTime">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>112</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="distance">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
